pkg/reconciler/instances/istio/README.md

```diff
@@ -1,17 +1,22 @@
-# istio-configuration reconciler
+# Istio Reconciler
 
 ## Overview
 
-The istio-configuration reconciler manages `istio-configuration` Kyma component.
+Istio Reconciler manages [Istio](https://github.com/kyma-project/kyma/tree/main/resources/istio-configuration) Kyma component. We support two latest minor Kyma 2.x releases and the `main` Kyma version.
 
-## Run Reconciler locally (Mac OS)
+## Prerequisites
 
-Follow these steps to run istio-configuration reconciler locally on your cluster:
+The Istio component requires `cluster-essentials` to be installed as a prerequisite.
 
-1. Export `KUBECONFIG` variable to point to your cluster.
+## Usage
+
+Follow these steps to run Istio Reconciler locally, on your cluster:
```

Comment:
```suggestion
Follow these steps to run Istio Reconciler locally, on your cluster:
```
I just spotted it now: did you mean `locally OR on your cluster`? 
If you meant a local cluster (like a k3d one), though technically it's correct, people usually mean a remote cluster when they say "on the cluster". Maybe it would be worth rephrasing it to something like "on your local machine"/"on your local setup" or even just "on your local k3d cluster" (here we kind of impose the use of k3d, but then again, it's a prerequisite for local Kyma installation, so maybe it's okay?).